pages/Javascript-API.md

```diff
@@ -99,8 +90,8 @@ scene.screenshot().then(function(screenshot) { window.open(screenshot.url); });
 #### `setActiveCamera(camera)`
 Sets the active camera to the camera specified by name, as named in the scene file.
 
-#### `setDataSource(name, source)`
-Loads a new `[data source](sources.md)` or replaces an existing one, and rebuilds the scene.
+#### `setDataSource(_string_ name, _object_ config)`
+Loads a new `source` object (see `[sources](sources.md)`), returning a Promise which fulfills when the `source` is loaded.
```

Comment:
Ah yes, anything in the backticks gets force-code-formatted. You can wrap a code-block in a link, but not vice-versa. So to get "code style text" inside a link, you have to make it like this:

[`source`](source.md)

Compare to the bulleted links at the top of https://mapzen.com/documentation/tangram/shaders/